README.md

```diff
@@ -5,10 +5,10 @@
 The Zigbee R23 add-on for the nRF Connect SDK provides support for developing Zigbee applications based on the third-party precompiled ZBOSS stack. This stack is included as the ZBOSS Zigbee R23 stack library. In combination with the nRF Connect SDK, the Zigbee R23 add-on allows for development of low-power connected solutions.
 
 ## Getting started
-To get started with Zigbee R23 add-on for nRF Connect SDK, follow [documentation](https://nrfconnect.github.io/ncs-zigbee/).
+To get started with Zigbee R23 add-on for nRF Connect SDK, follow [documentation](https://docs.nordicsemi.com/bundle?cluster=true&exclude_metadata_filter.field=display-type&exclude_metadata_filter.value=inline&labelkey=addon-zigbee-r23).
 
 ## Documentation
-Pre-build documentation is available [here](https://nrfconnect.github.io/ncs-zigbee).
+Pre-build documentation is available [here](https://docs.nordicsemi.com/bundle?cluster=true&exclude_metadata_filter.field=display-type&exclude_metadata_filter.value=inline&labelkey=addon-zigbee-r23).
```

Comment:
This and the following steps to build documentation might be unnecessary - now that the docs are hosted is there any scenario in which someone might want to build the docs themselves?